airflow/contrib/hooks/gcs_hook.py

```diff
@@ -568,6 +563,102 @@ def create_bucket(self,
                 'Bucket creation failed. Error was: {}'.format(ex.content)
             )
 
+    def insert_bucket_acl(self, bucket, entity, role, user_project):
+        # type: (str, str, str, str) -> None
+        """
+        Creates a new ACL entry on the specified bucket.
```

Comment:
Might want to link to a doc here and in related methods

Reply:
Done